CHANGELOG.md

```diff
@@ -32,6 +32,10 @@ Current Trunk
 - Add the ability to create multivalue Types in the C and JS APIs.
 - Remove named function types. They are replaced by `params` and `results` types
   local to each function.
+- Binaryen.js can now be compiled to Wasm using the `binaryen_wasm` target.
+  Unlike the JS variant, the Wasm variant requires asynchronously awaiting the
+  Wasm blob's instantiation and initialization before being usable, using the
+  `binaryen.ready` promise, e.g. `binaryen.ready.then(() => ...)`.
```

Comment:
I think we export as `Binaryen` (with a capital `B`)?

Reply:
Ah, yeah, the default is `Binaryen` when loading binaryen.js in the browser, that is just including the script. For the npm package I'm using `const binaryen = require("binaryen");` simply because that's the package name and somewhat standard to do. Perhaps it makes sense to unify both to lower case to match the package name?

Reply:
Oh, I see. Yeah, let's maybe standardize it on the lowercase name then. But maybe in a separate PR (with a changelog entry)?

So I think this is ready to land, except for the merge conflict.

Reply:
Sounds good! Conflict should be resolved if CI doesn't state otherwise.